hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/FileFormatUtilsForFileGroupReader.scala

```diff
@@ -124,4 +124,11 @@ object FileFormatUtilsForFileGroupReader extends SparkAdapterSupport {
       plan
     }
   }
+
+  def createStreamingDataFrame(sqlContext: SQLContext, relation: HadoopFsRelation, requiredSchema: StructType): DataFrame = {
+    val logRel = LogicalRelation(relation, isStreaming = true)
```

Comment:
nit: use English words to avoid abbreviation and confusion
```suggestion
    val logicalRelation = LogicalRelation(relation, isStreaming = true)
```